v4/integration/dydxStateManager/src/main/java/exchange/dydx/dydxstatemanager/AbacusStateManager.kt

```diff
@@ -197,7 +197,7 @@ class AbacusStateManager @Inject constructor(
             appConfigsV2.enableLogger = false
         }
 
-        appConfigsV2.staticTyping = featureFlags.isFeatureEnabled(DydxFeatureFlag.abacus_static_typing)
```

Comment:
this would turn it on in next OSS release? Or is there prod-gating code that I'm not seeing?

Reply:
Yeah.. it's turned on by default to simplify testing. If we want to cut a build without static typing, we'd need to switch it back off.